src/features/cross-chain/calculation-manager/providers/symbiosis-provider/symbiosis-cross-chain-trade.ts

```diff
@@ -160,6 +161,9 @@ export class SymbiosisCrossChainTrade extends EvmCrossChainTrade {
             crossChainTrade.to.blockchain
         );
         this.contractAddresses = crossChainTrade.contractAddresses;
+        this.promotions = crossChainTrade?.promotions
```

Comment:
this.promotions = crossChainTrade?.promotions || super.promotions;

Reply:
Done.